src/ResourceManager/Storage/Commands.Management.Storage/packages.config

```diff
@@ -1,6 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Storage" version="6.5.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Storage" version="7.1.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
```

Comment:
@blueww same comment about removing the added references since they are already referenced from a common project